i18n/pt.json

```diff
@@ -0,0 +1,59 @@
+{
+    "composeNewTweet": "Tweetar",
+    "navBookmarks": "Itens salvos",
+    "navDirectMessage": "Mensagens",
+    "navExplore": "Explorar",
+    "navHome": "Página inicial",
+    "navLists": "Listas",
+    "navNotifications": "Notificações",
+    "navProfile": "Perfil",
+    "statsFollowers": "Seguidores",
+    "statsFollowing": "Seguindo",
+    "statsLikes": "Curtidas",
+    "statsMoments": "Moments",
+    "statsTweets": "Tweets",
+    "translateTweet": "Traduzir Tweet",
+  
+    "bornDate": "Nascido(a) em $date$",
+    "bornYear": "Nascido(a) em $year$",
```

Comment:
I am not familiar with portuguese :c
When do you say `Nascido a` and when `Nascido em`?
I can attempt to write a little function to use the correct word when necessary, like I did here: #85

Reply:
Hello, thanks for being _quicc_.

``Nascido(a)`` is used officially by Twitter since born has two nouns in the portuguese language (a masculine and a feminine noun), so that's why the ``(a)`` is added to it.

Reply:
Oh okay, that makes it easier for me :D
I'm gonna merge it then.